src/Block/block.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Automattic\Chatrix\Block;
+
+function register() {
+	$block_path = realpath( plugin_dir_path( __FILE__ ) . '../../build-block' );
```

Comment:
If `npm run build:block-internal` was not run, this returns false which in turn causes the seemingly unrelated error message
```
Notice: Function WP_Block_Type_Registry::register was called incorrectly. Block type names must contain a namespace prefix. Example: my-plugin/my-custom-block-type Please see [Debugging in WordPress](https://wordpress.org/support/article/debugging-in-wordpress/) for more information. (This message was added in version 5.0.0.) in /Users/alex/Sites/localhost/wp/wp-includes/functions.php on line 5831
```

Reply:
The message is a bit weird indeed, because the first argument can either be the block name, or the path to `block.json`. If `block.json` doesn't exist `$block_path` is `false`, so that message is triggered.

I pushed a commit to only register the block if the directory exists.

Reply:
Maybe we could do it with or like `_doing_it_wrong()`? http://developer.wordpress.org/reference/functions/_doing_it_wrong/

Reply:
I looked into `_doint_it_wrong()` but that seems to be for internal WordPress code, as it requires a `$version` parameter for `The version of WordPress where the message was added.`. I might be wrong though.

I now pushed a commit that adds a site status test for when `block.json` is not found:

<img width="856" alt="Screenshot 2022-10-03 at 13 10 40" src="https://user-images.githubusercontent.com/550401/193574013-decc0b75-2762-4513-bac4-938ef9f3b193.png">

Doing you think this covers it?